frontend/.gitignore

```diff
@@ -21,3 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/ <reference types="react-scripts" />
+
```

Comment:
Sorry, I didn't mean for you to put the file's contents here. I meant for you to type out the file's name here. Simply write `src/react-app-env.d.ts` in this file. And put the lines that you copied over back into the original file.